src/jvm/main/org/jetbrains/kotlinx/lincheck/CTestStructure.java

```diff
@@ -49,13 +51,13 @@ private CTestStructure(List<ActorGenerator> actorGenerators, List<ParameterGener
      * Constructs {@link CTestStructure} for the specified test class.
      */
     public static CTestStructure getFromTestClass(Class<?> testClass) {
-        Map<String, OperationGroup> groupConfigs = new HashMap<>();
-        List<ActorGenerator> actorGenerators = new ArrayList<>();
-        List<Method> validationFunctions = new ArrayList<>();
-        List<Method> stateRepresentations = new ArrayList<>();
+        Map<String, OperationGroup> groupConfigs = new Object2ObjectOpenHashMap<>();
```

Comment:
Just a random suggestion. 
Maybe add type-aliases like `LincheckHashMap` and factory functions like `lincheckListOf()` and use them everywhere? 
I think such solution has several benefits:
1) The underlying collection implementations can be changed quite easily by changing type-aliases.
2) We would hide collections' implementation details in one place.
3) The name `Object2ObjectOpenHashMap` is quire verbose compared to something like `LincheckHashMap`.

Reply:
Oh, I see you did something similar in `CollectionUtils.kt`, but I guess there is no way to use type-aliases from Java : (

Reply:
Anyway, what about using `LincheckMutableHashMap` naming scheme instead of `MutableObjectToObjectMap`?

Reply:
I agree with you, and due to this limitation on type aliases in Java, the solution with factory-like top-level functions as `lincheckListOf()` looks pretty nice, especially taking into account your arguments above.  @ndkoval Do you still think it is redundant?

Reply:
I also like the flexibility it brings.

Reply:
Please put this discussion as a comment.